compiler+runtime/src/cpp/jank/read/lex.cpp

```diff
@@ -313,12 +325,52 @@ namespace jank::read
 
     static native_bool is_symbol_char(char32_t const c)
     {
-      return !std::iswspace(c) && !is_special_char(c)
+      return !std::iswspace(safe_cast_char32_t_to_int(c)) && !is_special_char(c)
```

Comment:
I don't think this is what we want. What we want is this:

* We have a `char32_t` and we want to check if it's a space (true or false)

With this change, we have this:

* We have a `char32_t` and we want to check if it's a space (true or false), but we throw an exception if it's larger than an int

Instead, let's just provide our own `iswspace` which takes a `char32_t`. The docs for it are here: https://en.cppreference.com/w/cpp/string/wide/iswspace Looks like a simple switch with 6 cases should work.

Reply:
updated to a static cast instead.